Website/src/rendering/src/components/Sponsors/SponsorInformation.tsx

```diff
@@ -1,29 +1,48 @@
 import { Field, RichText } from '@sitecore-jss/sitecore-jss-nextjs';
 import { ComponentProps } from 'lib/component-props';
 import SessionList from '../Sessions/SessionList';
-import { SESSIONS } from '../../models/mock-sessions';
+import { GraphQLSession } from 'src/types/session';
 
 export type SponsorInformationProps = ComponentProps & {
   fields: {
-    Description: Field<string>;
+    data: {
+      contextItem: {
+        description: Field<string>;
+        sessions: {
+          targetItems: GraphQLSession[];
+        };
+      };
+    };
   };
 };
 
-const SponsorInformation = (props: SponsorInformationProps): JSX.Element => (
-  <section className="section information-section">
-    <div className="section__content container">
-      <div className="information-grid">
-        <div className="main-col">
-          <div className="column-title">Sponsor history:</div>
-          <RichText className="rich-text" field={props.fields.Description} />
-        </div>
-        <div className="sidebar-col">
-          <div className="column-title">Sessions:</div>
-          <SessionList sessions={SESSIONS} showSpeakers={true} />
+const SponsorInformation = (props: SponsorInformationProps): JSX.Element => {
+  const SessionListSidebar =
+    props.fields.data.contextItem.sessions.targetItems.length > 0 ? (
+      <>
+        <div className="column-title">Sessions:</div>
+        <SessionList
+          sessions={props.fields.data.contextItem.sessions.targetItems}
+          showSpeakers={true}
+        />
+      </>
+    ) : (
+      <div className="column-title">There are no sessions available yet.</div>
```

Comment:
When there is no sessions, the message and its style is different than when there is no speaker on a session (Example: /sessions/Lunch). Could you adjust the message and style to have a similar experience on all type of pages?